### ISSUE TITLE
The tool and the action have different defaults, which is confusing

### ISSUE THREAD

**Issue by ulysses4ever** [andreasabel/fix-whitespace-action#3]
> With input `configfile` one can specify the configuration file (defaults to fix-whitespace.yaml).

I find the default confusing in a scenario when I try to repro it locally and I can't run the action. I run `fix-whitespace` manually, and, of course, I get different results if I forget that I need to specify the config manually.

In short, I think the action and the tool should have the same default, if possible. Perhaps, people will be unhappy if you remove the default value from the action. So, a less disruptive way would be to add the same default to the tool.

**Comment by andreasabel** [comment]
I suppose you refer to this setting in the action: https://github.com/andreasabel/fix-whitespace-action/blob/61d9b34fdd998f8e550631a5c0dfde0cfe75addd/action.yml#L13-L16
This corresponds to `fix-whitespace --check`.

I think when running as action, actually fixing the whitespace issues makes little sense, as you would need to commit them back to the repo to have effect.

When running on the cmdline, fixing seems to make more sense, see discussion in:
- https://github.com/agda/fix-whitespace/issues/28

**Comment by ulysses4ever** [comment]
No, I was reffering to a different parameter: `configfile`

https://github.com/andreasabel/fix-whitespace-action/blob/61d9b34fdd998f8e550631a5c0dfde0cfe75addd/action.yml#L9-L12

I can sware I observed different behavior between the tool and the action if you don't explicitly supply the `configfile`/`config`  parameter. But now I can see that the tool seem to have the same default as the action:

https://github.com/agda/fix-whitespace/blob/e04ea743f614960e3319a58ffae5a542c76a26b7/FixWhitespace.hs#L84-L86

And it was committed long ago.

So, I'm confused as to how I ended up opening this issue...

As for the default check v. fix, let's move to the other issue.
